lib/modules/auth/register/bloc/register_bloc.dart

```diff
@@ -0,0 +1,46 @@
+import 'dart:async';
+
+import 'package:bloc/bloc.dart';
```

Comment:
to get the validator class to your branch merge register-bloc development 
so
on register-bloc ==> git pull 
checkout to development ==> git pull
checkout to register-bloc ==> git pull
after that merge using 
git merge register-bloc development